driver/bpf/filler_helpers.h

```
@@ -807,22 +865,42 @@ static __always_inline int __bpf_val_to_ring(struct filler_data *data,
 				len = PPM_MAX_ARG_SIZE;
 
 			if (!data->curarg_already_on_frame) {
-				volatile unsigned long read_size = len;
+				volatile u16 read_size = len;
```

Is this okay or are we silently truncating invalid values to fit in the valid range?

---

This change came from the rebase.
It turns out len is already bounded to an upper value of PPM_MAX_ARG_SIZE at line 807/865 just above,
and PPM_MAX_ARG_SIZE is 65000, which fits into 16 bits.